makePoplogCommander.sh

```diff
@@ -1,9 +1,33 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
As we're in bash now, let's enable strict more
`set -euo pipefail`

Reply:
Fixed